Sama_Databricks_Connector_Tutorial.ipynb

```diff
@@ -31,7 +30,10 @@
     }
    },
    "source": [
-    "# Install Sama SDK and Databricks Connector"
+    "# Step 1: Requirements\n",
+    "**1.1 Installation requirements**\n",
+    "Before you begin, make sure you install PySpark and Sama\n",
+    "By runing the following cell you will download and install PySpark, Sama, and their dependencies."
```

Comment:
I think PySpark is installed by default in Databricks? Can you confirm?

Reply:
Updated